components/session-service/oidc/oidc.go

```diff
@@ -94,6 +98,14 @@ func New(cfg Config, retrySec int, certs *certs.ServiceCerts, l OIDCLogger) (Cli
 		log:      l}, nil
 }
 
+func (c *client) RefreshTokenValidator(refreshToken string) (*http.Response, error) {
+	temp := myurl
+	temp += "/tokenValid"
+	formData := url.Values{}
+	formData.Add("refresh_token", refreshToken)
+	return c.Client().PostForm(temp, formData)
+}
+
```

Comment:
@SanjuPal01 variable name myurl could be changed to clientURL and that can be a part of client struct and use it in this function as c.clientURL, also code could be refactored with readable variable names, avoid using temp.

Reply:
okay will do